src/benchmark/processes.py

```diff
@@ -194,6 +198,85 @@ def _get_benchmark_app_metric(self, metric_name):
                     return None
 
 
+class OpenVINOBenchmarkCppProcess(OpenVINOProcess):
+    def __init__(self, test, executor, log, cpp_benchmark_path, perf_hint=''):
+        super().__init__(test, executor, log)
+        self._benchmark_path = cpp_benchmark_path
```

Comment:
Are there any check anywhere that cpp_benchmark_path should be existing path to the file? I can't find it...

Reply:
Thanks, added pathlib check